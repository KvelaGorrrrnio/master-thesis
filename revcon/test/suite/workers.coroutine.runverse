test/suite/workers.coroutine.revcon:
[store]
  main: Ã˜
  ws: []

[stack]
[spawn main main end () | unresume main (ws) | despawn main main begin ()] <|> []

