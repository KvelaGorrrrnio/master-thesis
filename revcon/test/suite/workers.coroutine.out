test/suite/workers.coroutine.revcon:
[store]
  main: Ã˜
  ws: [worker (), worker ()]

[stack]
[spawn main main begin () | resume main (ws) | despawn main main end ()] <|> []

